CHANGELOG

```diff
@@ -1,3 +1,25 @@
+Changes in cclib-1.6.4
+
+Features:
+    * [GSOC2020] New method: bader calculates QTAIM partial charges (Minsik Cho)
+    * [GSOC2020] New methods: bickelhaupt, stockholder, hirshfeld and ddec calculate partial charges (Minsik Cho)
+    * [GSOC2020] New bridge to Horton (Minsik Cho)
+    * [GSOC2020] Support reading cube files in volume method (Minsik Cho)
+    * New bridge to Atomic Simulation Environment (Felipe S. S. Schneider)
+    * New bridge to PySCF (Amanda Dumi)
+    * New attribute dispersionenergies for molecular dispersion energy corrections (Theo Keane)
+    * New attribute vibfconsts for vibrational force constants (Chikashi Shinagawa)
+    * New attribute vibrmasses for vibrational reduced masses (Chikashi Shinagawa)
+    * Support t1_diagnostic in metadata for most parsers (Felipe S. S. Schneider)
```

Comment:
This was requested by Felipe but actually implemented by me.

Reply:
Ok, same response as above.

Reply:
Same.